package-lock.json

```diff
@@ -0,0 +1,893 @@
+{
```

Comment:
A brief description at the top here would be helpful

Reply:
This is automatically generated from NPM.